download_posts.py

```diff
@@ -7,7 +7,7 @@
 from auth import cookies, headers
 
 
-YEARS = range(2003, 2015)  # first to (last + 1)
+YEARS = range(1990, 2022)  # first to (last + 1)
```

Comment:
Should it be 2023 already?
```suggestion
YEARS = range(1990, 2023)  # first to (last + 1)
```